resources/assets/coffee/react/_components/friend-button.coffee

```diff
@@ -137,5 +137,6 @@ class @FriendButton extends React.PureComponent
     # - not viewing own card
     # - already a friend or can add more friends
     currentUser.id? &&
+      _.isFinite(@props.user_id) &&
```

Comment:
wait why does this not just do `@props.user_id?` just like the line right above it?

Reply:
oh wtf it parsed `null` (string) as `NaN` ಠ_ಠ